isaric/parsers/isaric-rapid.toml

```diff
@@ -443,22 +443,22 @@
 
 [visit.treatment_nsaid]
   description = "NSAID"
-  combinedType = "any"
+  combinedType = "firstNonNull"
 
   [[visit.treatment_nsaid.fields]]
-    field = "nsaid_cmoccur"
+    field = "overall_nsaid_cmyn"
     "ref" = "Y/N/NK"
-    description = "Non-steroidal anti-inflammatory (NSAID) on first day of admission?"
+    description = "Non-steroidal anti-inflammatory (NSAID)?"
 
   [[visit.treatment_nsaid.fields]]
     field = "daily_nsaid_cmoccur"
     "ref" = "Y/N/NK"
     description = "Non-steroidal anti-inflammatory (NSAID)?"
 
   [[visit.treatment_nsaid.fields]]
-    field = "overall_nsaid_cmyn"
+    field = "nsaid_cmoccur"
     "ref" = "Y/N/NK"
-    description = "Non-steroidal anti-inflammatory (NSAID)?"
+    description = "Non-steroidal anti-inflammatory (NSAID) on first day of admission?"
 
 [visit.treatment_other]
```

Comment:
The mapping for 'treatment_other' was also supposed to be a list. In my understanding a list of 'all' other treatments not captured in the schema. This is yet to be added to the parser ..